app/views/users/show.html.erb

```diff
@@ -0,0 +1,92 @@
+<div class="container">
+  <div class="row">
+    <div class="col-sm-12">
+      <h1>Users Profile</h1>
+      <h2><%= @user.name %></h2>
+      <p>this is a nice profile</p>
+      <hr>
+    </div>
+    <div class="col-sm-12 col-md-6">
+
+      <h3>Events created</h3>
+      <br>
+      <h4>Upcoming Events</h4>
+      <table class="table table-striped table-hover">
+
+        <thead>
+          <tr>
+            <th scope="col">#</th>
+            <th scope="col">Name</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @user.events.upcoming.each do |event| %>
+            <tr class="cursor-pointer" onclick="location.href = '<%= event_path(event.id) %>';">
+              <td><%= event.id %></td>
+              <td><%= event.name %></td>
+            </tr>
+          <% end  %>
+        </tbody>
+      </table>
+
+      <h4>Previous Events</h4>
+      <table class="table table-striped table-hover">
+
+        <thead>
+          <tr>
+            <th scope="col">#</th>
+            <th scope="col">Name</th>
+          </tr>
+        </thead>
+        <tbody>
+          <% @user.events.past.each do |event| %>

```

Comment:
- [ ] let's make the views as simple as possible and avoid any kind of logic, for example this could be @prev_events instantiated in the show action, do that every time is possible.

Reply:
Thanks for the comments, I'll work on it, about my partner, thanks for the concern but we agreed to work individually in this next projects.

Reply:
Hi @men32z  was it a personal decision or was this suggested by student success?